evadb/evadb.yml

```diff
@@ -15,17 +15,15 @@ executor:
 
   gpu_ids: [0]
 
-storage:
-  tmp_dir: ""
-  s3_download_dir: ""
-
 server:
   host: "0.0.0.0"
   port: 8803
   socket_timeout: 60
 
 experimental:
-  ray: False 
+  ray: False
+  # Set the postgres db uri to connect to in 'catalog_database_uri' 
+  use_postgres_backend: False
```

Comment:
Do we need this flag? We can infer it directly from the `catalog_database_uri`. We can avoid asking users for an extra thing.

Reply:
Done, removed it.